auth-source-pass.el

```diff
@@ -197,7 +196,15 @@ CONTENTS is the contents of a password-store formatted file."
 
 Disambiguate between user provided inside HOST (e.g., user@server.com) and
 inside USER by giving priority to USER.  Same for PORT."
-  (apply #'auth-source-pass--find-match-unambiguous (auth-source-pass--disambiguate host user port)))
+  (seq-some (lambda (a-host)
```

Comment:
Can you please rename `a-host` to just `host` and change the function parameter name to `hosts`? Also, can you please change the docstring to make it clear that `hosts` is either a list or a string.

Reply:
I will check and correct it.

Reply:
Updated parameter names and docstrings in `auth-source-pass--build-result` and `auth-source-pass--find-match`.